File: C++/1158.cpp
#include<iostream>
using namespace std;

int main(){

    int entrada;
    int x, y;
    int soma, total = 0;

    cin >> entrada;

    for(int i = 0; i < entrada; i++){
        cin >> x >> y;
        soma = x;
        total = 0;
        if(soma % 2 == 0){
            soma = soma + 1;
        }
        for(int j = 0; j < y; j++){
            total = total + soma;
            soma = soma + 2;
        }
        cout << total << endl;
    }

    return 0;
}

File: JavaScript/1158.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');

let entrada = parseInt(lines[0])
let x, y, valores;
let soma, total = 0;

for(let i = 1; i <= entrada; i++){
    valores = lines[i].split(' ')

    x = parseInt(valores.shift())
    y = parseInt(valores.shift())

    soma = x;
    if(soma % 2 == 0){
        soma = soma + 1;
    }
    total = 0;
    for(let j = 0; j < y; j++){
        total = total + soma;
        soma = soma + 2;
    }
    console.log(total);

}